### DISCUSSION TITLE
Scope of GH-55: Create new DB Type for Amplify Operations

### DISCUSSION THREAD

**Discussion by johnnyhchen** [gp-seattle/rms-app#138] [Backend Questions]
Is enabling DataStore all I need to do for GH-55 (task 3 here: https://github.com/gp-seattle/rms-app/wiki/AWS-Amplify-Project:-Datastore)? What does **“Create a new DB type for Amplify operations”** mean?

**Comment by yauj** [comment]
No. It's a slightly different thing. Do you need to make code changes to enable Amplify Datastore?

**Reply by johnnyhchen** [reply]
No code changes needed to enable DataStore. It's a few command line prompts that changed a few cloud formation/json templates. Main thing I think is that you need to enable conflict detection.

What would creating a new database type be then?
